pkgs/tools/system/auto-epp/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, pkgs, lib , fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "auto-epp";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "jothi-prasath";
+    repo = "auto-epp";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-7sI8K+7ZAdzBN/XOzYQQZ1f9t+fFo6fcXYzX6abNyQ8=";
+  };
+
+  buildInputs = [ pkgs.python3 ];
```

I'm not entirely convinced this is supposed to be a normal `buildInputs`. I think maybe it should be wrapped somehow?

```suggestion
  buildInputs = [ python3 ];
```